conjure-java-undertow-runtime/src/test/java/com/palantir/conjure/java/undertow/runtime/BearerTokenLoggingTest.java

```diff
@@ -69,7 +71,10 @@ public void before() {
         delegateRunnable.set(null);
         exchange = HttpServerExchanges.createStub();
         exchange.setRequestMethod(Methods.GET);
-        handler = new BearerTokenLoggingHandler(delegate);
+        handler = new LoggingContextHandler(httpServerExchange -> {
+            Auth.header(httpServerExchange);
```

Comment:
From the method signature, it's a tiny bit surprising that this has the side effect of writing to the MDC too... do you think it's worth trying to get the acronym 'MDC' in there somewhere? Either `MdcAuth.header()` or `Auth.headerToMdc`?

Reply:
I'd prefer to keep the current method names -- in the future we may want to code-gen a handler to extract auth tokens on an IO thread, and update this method to do a simple attachment lookup. This way we avoid expanding the API based on implementation details.